crates/cli/src/subcommands/publish.rs

```diff
@@ -39,6 +39,13 @@ pub fn cli() -> clap::Command {
                 .short('p')
                 .help("The system path (absolute or relative) to the module project")
         )
+        .arg(
+            Arg::new("wasm_file")
+                .value_parser(clap::value_parser!(PathBuf))
+                .long("wasm-file")
+                .short('w')
+                .help("The system path (absolute or relative) to the wasm file we should publish, instead of building the project."),
```

Comment:
As ingvar stated in Discord, the `spacetime generate` command also has this wonkyness of accepting `--project-path`, could we update it there as well? Thanks Zeke 🙏

Reply:
Thank you! Brought `spacetime publish` in line with `spacetime generate` and `spacetime build`.